src/pages/MyEarnings/ClassicPools/SinglePool/Position.tsx

```diff
@@ -0,0 +1,272 @@
+  width: 100%;
+  height: 0;
+  border-bottom: 1px solid ${({ theme }) => theme.border};
+`
+
+type ColumnProps = {
+  label: string
+  value: React.ReactNode
+}
+const Column: React.FC<ColumnProps> = ({ label, value }) => {
+  const theme = useTheme()
+  return (
+    <Flex
+      sx={{
+        flexDirection: 'column',
+        gap: '8px',
+        fontWeight: 500,
+        fontSize: '20px',
+        lineHeight: '24px',
+        color: theme.text,
+      }}
+    >
+      <Text
+        as="span"
+        sx={{
+          fontWeight: 500,
+          fontSize: '12px',
+          lineHeight: '16px',
+          color: theme.subText,
+        }}
+      >
+        {label}
+      </Text>
+
+      {value}
+    </Flex>
+  )
+}
+
+const ColumnForMobile: React.FC<ColumnProps & { align?: 'left' | 'right' }> = ({ label, value, align = 'left' }) => {
+  const theme = useTheme()
+  return (
+    <Flex
+      sx={{
+        flexDirection: 'column',
+        gap: '4px',
+        fontWeight: 500,
+        fontSize: '16px',
+        lineHeight: '20px',
+        color: theme.text,
+        alignItems: align === 'left' ? 'flex-start' : 'flex-end',
+      }}
+    >
+      <Text
+        as="span"
+        sx={{
+          fontWeight: 500,
+          fontSize: '12px',
+          lineHeight: '16px',
+
+          color: theme.subText,
+        }}
+      >
+        {label}
+      </Text>
+
+      {value}
+    </Flex>
+  )
+}
+
+type Props = {
+  currency0: Currency | undefined
+  currency1: Currency | undefined
+  poolAddress: string
+  chainId: ChainId
+  userLiquidity: UserLiquidityPosition | undefined
+}
+const Position: React.FC<Props> = ({ chainId, userLiquidity, currency0, currency1, poolAddress }) => {
+  const theme = useTheme()
+  const upToExtraSmall = useMedia(`(max-width: ${MEDIA_WIDTHS.upToExtraSmall}px)`)
+  const networkInfo = NETWORKS_INFO[chainId]
+  const totalSupply = useTotalSupplyV2(chainId, poolAddress)
+
+  const myLiquidityBalance = userLiquidity ? getMyLiquidity(userLiquidity, '0') : '--'
+  const { decimals, value: rawBalance } = useTokenBalance(poolAddress, chainId)
+  const balance = Number(
+    new Fraction(rawBalance.toString(), JSBI.exponentiate(JSBI.BigInt(10), JSBI.BigInt(decimals))).toFixed(8),
+  )
+
+  // TODO: check native currencies
+  const currency0Slug = ''
+  const currency1Slug = ''
+
+  if (upToExtraSmall) {
+    return (
+      <Flex
+        sx={{
+          flexDirection: 'column',
+          gap: '16px',
+        }}
+      >
+        <Flex justifyContent={'space-between'}>
+          <ColumnForMobile label={t``} value={formatValue('1234567')} />
+          <ColumnForMobile align="right" label={t`APR`} value={formatPercent('12.34')} />
+        </Flex>
+
+        <MobileSeparator />
+
+        <Flex justifyContent={'space-between'}>
+          <ColumnForMobile label={t`VOLUME (24H)`} value={formatValue('1234567')} />
+          <ColumnForMobile align="right" label={t`FEES (24H)`} value={formatValue('1234567')} />
```

Comment:
is this file unused?

Reply:
Yeah, currently Classic pools is not used yet

Reply:
So remove this file? Or will it being used later?

Reply:
We can keep. Classic pools is hidden in the UI